commons/jdk22/src/main/java/org/infinispan/commons/jdk22/logging/Jdk22Log.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.commons.jdk22.logging;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.MessageLogger;
+import org.jboss.logging.annotations.ValidIdRange;
+import org.jboss.logging.annotations.ValidIdRanges;
+
+/**
+ * Keep in sync with the Log class in the commons package. Anything added here must also be placed there and commented.
+ * @api.private
+ */
+@MessageLogger(projectCode = "ISPN")
+@ValidIdRanges({
+      @ValidIdRange(min = 901, max = 1000),
```

Comment:
Asking because of the range overlap, I guess depending on infinispan-commons isn't possible?

Reply:
why do we need a specific logger ?

Reply:
Correct, if you try to import infinispan-commons you get a cycylical reference dependency. So I had to include the Logger in the new module.